src/hdf5_interface.F90

```diff
@@ -213,12 +218,11 @@ end subroutine file_close
 
   subroutine get_groups(object_id, names)
     integer(HID_T), intent(in)  :: object_id
-    character(len=255), allocatable, intent(out) :: names(:)
+    character(len=150), allocatable, intent(out) :: names(:)
```

Comment:
Oh to be consistent with some code which uses this and to stop getting the warnings about truncation.  I figured even 150 was overkill, so I just matched it.